ui-grid/ui-grid.d.ts

```diff
@@ -3767,7 +3767,7 @@ declare module uiGrid {
          */
         sortCellFiltered?: boolean;
         /** Algorithm to use for sorting this column */
-        sortingAlgorithm?: (a: any, b: any) => number;
+        sortingAlgorithm?: (a: any, b: any, rowA: any, rowB: any, direction: any) => number;
```

Comment:
the `direction` argument isn't really an `any`, it will only be one of two values: `uiGridConstants.ASC` (`'asc'`) or `uiGridConstants.DESC` (`'desc'`).  Since TS 1.8 isn't released yet, a better type annotation for `direction` today would be `string`, but once TS 1.8 is released, it should probably be `'asc' | 'desc'`.
Also, a better type for `rowA` and `rowB` would be `IGridRowOf<TEntity>`.